README.md

```diff
@@ -0,0 +1,795 @@
+| ARM      | 2022-05-01         | /subscriptions/`subscriptionId`/providers/Microsoft.Network/virtualNetworks                                                            |
+| ARM      | 2020-06-01         | /subscriptions/`subscriptionId`/providers/Microsoft.Resources/tags/default                                                             |
+| ARM      | 2018-06-01         | /subscriptions/`subscriptionId`/providers/Microsoft.Security/pricings                                                                  |
+| ARM      | 2020-01-01         | /subscriptions/`subscriptionId`/providers/Microsoft.Security/securescores                                                              |
+| ARM      | 2020-01-01-preview | /subscriptions/`subscriptionId`/providers/Microsoft.Security/securityContacts                                                          |
+| ARM      | 2019-10-01         | /subscriptions/`subscriptionId`/providers                                                                                              |
+| ARM      | 2021-04-01         | /subscriptions/`subscriptionId`/resourcegroups                                                                                         |
+| ARM      | 2023-07-01         | /subscriptions/`subscriptionId`/resources                                                                                              |
+| ARM      | 2020-01-01         | /subscriptions                                                                                                                         |
+| ARM      | 2020-01-01         | /tenants                                                                                                                               |
+
+## Integrate with AzOps
+
+Did you know you can run AzOps from Azure DevOps? Check [AzOps Accellerator](https://github.com/Azure/AzOps-Accelerator).
+You can integrate Azure Governance Visualizer (same project as AzOps).
+
+```yaml
+  pipelines:
+    - pipeline: 'Push'
+      source: 'AzOps - Push'
+      trigger:
+        branches:
+          include:
+            - master
+```
+
+## Integrate PSRule for Azure
+
+__Pausing 'PSRule for Azure' integration__. Azure Governance Visualizer leveraged the Invoke-PSRule cmdlet, but there are certain [resource types](https://github.com/Azure/PSRule.Rules.Azure/blob/ab0910359c1b9826d8134041d5ca997f6195fc58/src/PSRule.Rules.Azure/PSRule.Rules.Azure.psm1#L1582) where also child resources need to be queried to achieve full rule evaluation.
+
+Let´s use [PSRule for Azure](https://azure.github.io/PSRule.Rules.Azure) and leverage over 260 pre-built rules to validate Azure resources based on the Microsoft Well-Architected Framework (WAF) principles.
+PSRule for Azure is listed as [security monitoring tool](https://docs.microsoft.com/en-us/azure/architecture/framework/security/monitor-tools) in the Microsoft Well-Architected Framework.
```

Comment:
I don't think it is anymore (and this link is not updated).  If it should be, please open a PR to remediate.